justspacesapp/views.py

```diff
@@ -30,3 +35,23 @@ class SurveyList(ListView):
     model = FormEntry
     template_name = "survey_list.html"
     context_object_name = 'surveys'
+
+class Signup(FormView):
```

Comment:
Do we need to expose a URL for Signup in the sidebar?

Reply:
yes!